### DISCUSSION TITLE
空数据的单元格，中划线可以去掉吗

### DISCUSSION THREAD

**Discussion by KuduroJS** [antvis/S2#2406] [Q&A] [answered]
https://s2.antv.vision/examples/basic/table/#table

比如这个demo的成本一列

**Comment by lijinke666** [comment, marked as answer]
多看看文档吧, 多搜索, 实在找不到再提问, https://s2.antv.antgroup.com/api/general/s2-options#:~:text=%E5%8D%95%E5%85%83%E6%A0%BC%E4%BF%A1%E6%81%AF-,placeholder,-string%20%7C%20(meta%3A%20Record

```diff
const s2Options = {
  width: 600,
  height: 480,
  showSeriesNumber: true,
+ placeholder: ""
};
```

![image](https://github.com/antvis/S2/assets/21015895/e09717b6-0c33-4e5a-8123-59e01af60e7a)


> 另外, 官网请使用 https://s2.antv.antgroup.com/   [s2.antv.vision](https://s2.antv.vision/examples/basic/table/#table) 这个域名废弃了

**Reply by KuduroJS** [reply]
好的，谢谢
